app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is currently a class variable, which means it is shared across all instances of the `Animal` class. This can lead to unintended behavior, as all animals will share the same list. Consider making `alive` an instance variable instead if your intention is to have each animal manage its own list of living animals.